stormpath/src/test/java/com/stormpath/sdk/StormpathConfigurationTest.java

```diff
@@ -0,0 +1,45 @@
+package com.stormpath.sdk;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class StormpathConfigurationTest {
+
+    @Test(expected = IllegalStateException.class)
+    public void exceptionThrownIfBaseUrlNotSet() throws Exception {
+        new StormpathConfiguration.Builder().build();
+    }
+
+    @Test
+    public void pathsStartWithSlash() throws Exception {
+        StormpathConfiguration stormpath = new StormpathConfiguration.Builder()
+                .baseUrl("https://example.com")
```

Comment:
I set this API url in the usage docs and test https://github.com/stormpath/stormpath-sdk-android/commit/acc1765fcc077fab64f239bc2c103b8c68ac69fe. Maybe @edjiang knows if the version should be enforced?